lms/static/scripts/frontend_apps/utils/google-picker-client.ts

```diff
@@ -183,7 +194,7 @@ export class GooglePickerClient {
       }
     }
 
-    const view = new pickerLib.View(pickerLib.ViewId.DOCS);
+    const view = new pickerLib.DocsView(pickerLib.ViewId.DOCS);
```

Comment:
This did work previously, but `View` is actually meant to be an abstract base class and `DocsView` is what should be instantiated directly. See https://developers.google.com/drive/picker/reference#docs-view.